static/app/components/replays/videoReplayer.tsx

```diff
@@ -297,8 +309,16 @@ export class VideoReplayer {
 
     // No more segments
     if (nextIndex >= this._attachments.length) {
+      if (this.getCurrentTime() < this._durationMs) {
+        // If we're at the end of a segment, but there's a gap
+        // at the end, force the replay to play until the end duration
+        // rather than stopping right away.
+        this._timer.addNotificationAtTime(this._durationMs, () => {
+          this.stopReplay();
+        });
+        return;
+      }
```

Comment:
Is this tested?

Reply:
https://github.com/getsentry/sentry/pull/69074#discussion_r1569399619